nodebuilder/p2p/network.go

```diff
@@ -50,6 +62,28 @@ func (n Network) String() string {
 	return string(n)
 }
 
+// ChainIDFromNetwork parses the ChainID from the Network name.
+func ChainIDFromNetwork(net Network) ChainID {
+	if !strings.Contains(net.String(), "da") {
```

Comment:
Doesn't this break if there's "da" in chain id, such as darkroast-3-da-x?

Reply:
Yes this does and I realised that earlier, but we can't have a special separator be `/` as directory creation will break.

We can use `:` or `.` potentially.